plutus-ledger-api/src/PlutusLedgerApi/Common/ProtocolVersions.hs

```diff
@@ -53,6 +54,16 @@ alonzoPV = ProtocolVersion 5 0
 vasilPV :: ProtocolVersion
 vasilPV = ProtocolVersion 7 0
 
--- | Chang era is not yet published. It will probably be introduced in protocol version 8.0
-changPV :: ProtocolVersion
-changPV = ProtocolVersion 8 0
+-- | Protocol version 8.0 was the Valentine intra-era HF
+valentinePV :: ProtocolVersion
+valentinePV = ProtocolVersion 8 0
+
+-- | This is a placeholder for when we don't yet know what protocol
+-- version will be used for something. It's a very high protocol
+-- version that should never appear in reality.
+--
+-- We should not assign names to future protocol versions until it's
+-- confirmed that they are correct, otherwise we could accidentally
+-- associate something with the wrong protocol version.
+futurePV :: ProtocolVersion
+futurePV = ProtocolVersion maxBound 0
```

Comment:
I guess because this isn't really a "top bound" for protocol versions, it's a fake protocol version. Same thing as with `Version`, really. Perhaps again we just want a `knownProtocolVersions :: Set ProtocolVersion` value.

Reply:
I put in `knownPVs`